### PR TITLE
[DOCS] Amends the footnotes in the ML book

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR reviews and amends the footnotes in the ML book to make them less verbose.

### Preview

[Classification example](http://stack-docs_1028.docs-preview.app.elstc.co/guide/en/machine-learning/master/flightdata-classification.html)
[Regression example](http://stack-docs_1028.docs-preview.app.elstc.co/guide/en/machine-learning/master/flightdata-regression.html)
[Language identification](http://stack-docs_1028.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-lang-ident.html)

### CODE REVIEW

File: docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc [hunk 1]
```diff
@@ -118,12 +118,14 @@ POST _ingest/pipeline/_simulate
 ----------------------------------
 //NOTCONSOLE
 
-<1> The ID of the {lang-ident} trained model.
-<2> Indicates that only the top five languages (that is to say, the ones with the highest probability) are reported.
-In this example, 5 classes (in this case, languages) with the 
-highest probability will be reported.
-<3> The source object that contains the text to identify.
+<1> ID of the {lang-ident-cap} trained model.
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc:121]:
```suggestion
<1> ID of the {lang-ident} trained model.
```

File: docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc [hunk 2]
```diff
@@ -351,15 +350,16 @@ The snippet below shows a part of a document with the annotated results:
...
-<2> The probability is a value between 0 and 1. The higher the number, the more 
-confident the model is that the data point belongs to the named class. In this 
-example, `false` has a `class_probability` of 0.91 while `true` has only 0.08, 
-so the prediction will be `false`.
-<3> The `class_score` is a function of the probability. It is chosen so that the 
-decision to assign the data point to the class with the highest score maximizes 
-the minimum recall of any class.
+`class_score` for each class. 
+
+The `top_classes` object contains the predicted  classes with the highest
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc:355]:
```suggestion
The `top_classes` object contains the predicted classes with the highest 
```

File: docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc [hunk 3]
```diff
@@ -351,15 +350,16 @@ The snippet below shows a part of a document with the annotated results:
...
-the minimum recall of any class.
+`class_score` for each class. 
+
+The `top_classes` object contains the predicted  classes with the highest 
+scores. The `class_probability` is a value between 0 and 1. The higher the 
+number, the more confident the model is that the data point belongs to the named 
+class. In the example above, `false` has a `class_probability` of 0.91 while 
+`true` has only 0.08, so the prediction will be `false`. The `class_score` is a 
+function of the probability. It is chosen so that the decision to assign the 
+data point to the class with the highest score maximizes the minimum recall of
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc:361]: > ... maximizes the minimum recall...

Have we covered recall in this example? If not, this sentence might be too advanced for this context.

File: docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc [hunk 4]
```diff
@@ -378,8 +378,8 @@ POST _ml/data_frame/_evaluate
 
 <1> The destination index which is the output of the {dfanalytics-job}.
 <2> Calculate the training error by evaluating only the training data.
-<3> The field that contains the actual (ground truth) value.
-<4> The field that contains the predicted value.
+<3> Field containing the actual (ground truth) value.
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc:381]: > Field containing

I'm not sure I understand how this change is an improvement.

File: docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc [hunk 5]
```diff
@@ -118,12 +118,14 @@ POST _ingest/pipeline/_simulate
...
 
-<1> The ID of the {lang-ident} trained model.
-<2> Indicates that only the top five languages (that is to say, the ones with the highest probability) are reported.
-In this example, 5 classes (in this case, languages) with the 
-highest probability will be reported.
-<3> The source object that contains the text to identify.
+<1> ID of the {lang-ident} trained model.
+<2> Specifies the number of languages to report by descending order of 
+probability.
+<3> Source object containing the text to identify.
```
Comment (lcawl) [inline docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc:124]: > Source object containing...

Likewise, I'm not sure this is a clearer wording.

File: docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc (hunk 4 above)
Comment (szabosteve) [reply docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc:381]: @lcawl I tried to achieve shorter sentences with less complicated structures. I thought that this way the sentence is less conversational. Do you prefer to change it back?

File: docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc (hunk 5 above)
Comment (szabosteve) [reply docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc:124]: As above.

File: docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc (hunk 4 above)
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc:381]:
```suggestion
<3> The field that contains the actual (ground truth) value.
```

File: docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc [hunk 6]
```diff
@@ -378,8 +378,8 @@ POST _ml/data_frame/_evaluate
...
+<4> Field containing the predicted value.
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/flightdata-regression.asciidoc:382]:
```suggestion
<4> The field that contains the predicted value.
```

File: docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc (hunk 5 above)
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/ml-lang-ident.asciidoc:124]:
```suggestion
<3> The source object that contains the text to identify.
```

**Comment by szabosteve** [comment]
**Backport commits:**
7.x: 51ce410
7.8: 3182b0e

**Pull request merged**